File: minwook/8주차/문자열_다루기_기본.java
class Solution {
    public boolean solution(String s) {
        if(s.length()==4 || s.length()==6)
            return s.matches("\\d+"); // \\d+ 숫자에 해당하는 문자를 1개 이상 포함하는지 검사하는 정규표현식
        else
            return false;
    }
}

File: jeonghyeon/8week/문자열 다루기 기본.js
function solution(s) {
    if(s.length !== 4 && s.length !== 6) {
        // 길이가 4나 6이 아니면 false
        return false;
    }
    for(let i = 0; i < s.length; i++) {
        if(isNaN(s[i])) { // 문자열에 숫자가 있나 확인 Not a Number
            return false;
        }
    }
    return true;
}